server/@types/express/index.d.ts

```diff
@@ -30,6 +30,7 @@ declare module 'express-session' {
     updateGoalForm: UpdateGoalForm
     prisonerListSortOptions: string
     pageFlowQueue: PageFlowQueue
+    prisonerFunctionalSkills: FunctionalSkills
```

Comment:
Add the new prisonerFunctionalSkills property of type FunctionalSkills to the session